metadata-service/plugin/src/main/java/com/datahub/plugins/common/Validator.java

```diff
@@ -0,0 +1,46 @@
+package com.datahub.plugins.common;
+
+import java.util.HashSet;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+import org.apache.commons.lang3.StringUtils;
+
+
+public class Validator {
```

Comment:
Should we call this ConfigValidationUtils?

Reply:
Please respond

Reply:
Done